src/monitoring_service/blockchain.py

```diff
@@ -34,6 +38,12 @@ def create_channel_event_topics() -> List:
     ]
 
 
+def create_monitoring_service_topics() -> List:
```

Comment:
If we don't reuse this anywhere, we can also put `[None]` inline and remove the separate function.

Reply:
I already opened #56 for that, let's handle that there.